### PR TITLE
Fixed a few leaks in /PC/bdist_wininst/install.c

### PR DESCRIPTION
**Pull request by tiagofilipesilva**
This trivial commit fixes a few memory leaks in /PC/bdist_wininst/install.c

### CODE REVIEW

File: PC/bdist_wininst/install.c [hunk 1]
```diff
@@ -2553,8 +2565,10 @@ int DoUninstall(int argc, char **argv)
             lines_buffer_size += 10;
             lines = (char **)realloc(lines,
                                      sizeof(char *) * lines_buffer_size);
-            if (!lines)
+            if (!lines) {
```
Comment (serhiy-storchaka) [inline PC/bdist_wininst/install.c:2568]: The memory pointed by the old value of `lines` is leaked if `realloc()` returns NULL.

And seems `lines` is leaked in the code below.

**Review by serhiy-storchaka — COMMENTED** [review]
There are other leaks in this code.

**Review by serhiy-storchaka — CHANGES_REQUESTED** [review]
Please fix a leak in `realloc()`.

**Comment by vstinner** [comment]
I removed the " needs backport to 3.6" label, the 3.6 branch no longer accept bugfixes (only security fixes are accepted): https://devguide.python.org/#status-of-python-branches

**Comment by csabella** [comment]
@serhiy-storchaka, it appears that the original creator of this pull request has abandoned the issue.  Since this concerns refleaks, should this PR be closed a new one opened to address the problems?  Thanks.

**Comment by vstinner** [comment]
> @serhiy-storchaka, it appears that the original creator of this pull request has abandoned the issue. Since this concerns refleaks, should this PR be closed a new one opened to address the problems? Thanks.

The author signed the CLA. You should try to find someone else to recreate this PR and fix the issue identified by @serhiy-storchaka.

**Comment by csabella** [comment]
Created bpo-36901 for this pull request.

**Pull request closed without merging**